utils/gha-runners/get-system-info.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2022, Intel Corporation
+
+#
+# get-system-info.sh - Script for printing system info
+#
+
+function system_info {
+	echo "********** system_info **********"
+	cat /etc/os-release | grep -oP "PRETTY_NAME=\K.*"
+	uname -r
+	echo "libndctl: $(pkg-config --modversion libndctl || echo 'libndctl not found')"
+	echo "valgrind: $(pkg-config --modversion valgrind || echo 'valgrind not found')"
```

Comment:
You should use valgrind from pmem/valgrind (3.18)
I can see in the log valgrind 3.17.